extensions/azure/azure-test/src/testFixtures/java/org/eclipse/dataspaceconnector/azure/testfixtures/CosmosTestClient.java

```diff
@@ -0,0 +1,107 @@
+package org.eclipse.dataspaceconnector.azure.testfixtures;
+
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import org.eclipse.dataspaceconnector.spi.EdcException;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateFactory;
+import java.util.List;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+
+import static java.io.File.separator;
+import static java.lang.String.format;
+import static org.eclipse.dataspaceconnector.common.configuration.ConfigurationFunctions.propOrEnv;
+
+public interface CosmosTestClient {
+
+    static CosmosClient createClient() {
+        var cosmosKey = propOrEnv("COSMOS_KEY", null);
+        if (cosmosKey != null) {
+            return clientForCi(cosmosKey);
+        } else {
+            return localClient();
+        }
+    }
+
+    private static CosmosClient clientForCi(String cosmosKey) {
+        return new CosmosClientBuilder()
+                .key(cosmosKey)
+                .preferredRegions(List.of("westeurope"))
+                .endpoint("https://cosmos-itest.documents.azure.com:443/")
+                .buildClient();
+    }
+
+    private static CosmosClient localClient() {
+        try {
+            var endpoint = "https://127.0.0.1:8081";
+
+            var keystorePath = System.getProperty("java.home") + separator + "lib" + separator + "security" + separator + "cacerts";
+            var keystore = KeyStore.getInstance(KeyStore.getDefaultType());
+
+            var keystorePassword = "changeit".toCharArray();
+            keystore.load(new FileInputStream(keystorePath), keystorePassword);
+
+            var certificate = getCertificateFrom(endpoint);
+            keystore.setCertificateEntry("cosmosdb", certificate);
+
+            try (var output = new FileOutputStream(keystorePath)) {
+                keystore.store(output, keystorePassword);
+            }
+
+            return new CosmosClientBuilder()
+                    .key("C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==")
```

Comment:
document where this is coming from?

Reply:
I wasn't able to find something on the official documentation, but it can be found on the emulator github repository, though there's no mentions to the fact that's a fixed value. Added that link as referral